### PR TITLE
Final Permissions Cleanup

### PR DESCRIPTION
**Pull request by vkrmis**
**Story card:** [ch1546](https://app.clubhouse.io/simpledotorg/story/1546/final-cleanup-of-old-permissions)

## This addresses

- [x] Delete UserPermissions model and table
- [x] Remove migration scripts
- [x] Remove `permissions.rb` file

### CODE REVIEW

File: doc/adding-new-user-permissions.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## **This permission model is deprecated** _(October 2020)_
```
Comment (harimohanraj89) [inline doc/adding-new-user-permissions.md:1]: I think you don't need to bold it if it's an H2 😅

File: doc/arch/010-user-permissions.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # User Permissions
+## **This permission model is deprecated** _(October 2020)_
```
Comment (harimohanraj89) [inline doc/arch/010-user-permissions.md:2]: Can we say it has been replaced by [link to Epic/PRD]?

**Review by harimohanraj89 — APPROVED** [review]
This look beautiful. Few comments from me.

PS what do devs need to do before this is merged? Run `rails db:migrate:with_data` locally?

**Comment by vkrmis** [comment]
> PS what do devs need to do before this is merged? Run `rails db:migrate:with_data` locally?
✅

File: doc/adding-new-user-permissions.md (hunk 1 above)
Comment (vkrmis) [reply doc/adding-new-user-permissions.md]: 😎

File: doc/arch/010-user-permissions.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
 # User Permissions
+## This permission model is deprecated (October 2020)
```
Comment (kitallis) [inline doc/arch/010-user-permissions.md:2]: Let's not do this in the ADRs. ADRs are meant to be immutable.

Comment (vkrmis) [reply doc/arch/010-user-permissions.md:2]: Hmm, how can we indicate this is no longer relevant though? Don't want to mislead someone looking at these in the future.

Comment (harimohanraj89) [reply doc/arch/010-user-permissions.md:2]: We could [write an ADR next week](https://app.clubhouse.io/simpledotorg/story/1227/write-an-adr-to-document-how-the-new-permissions-system-works) 😄

Comment (vkrmis) [reply doc/arch/010-user-permissions.md:2]: Would still want to say this, and link to that one instead of the PRD...

Comment (vkrmis) [reply doc/arch/010-user-permissions.md:2]: Looks like an ADR's `status` can be updated: using `deprecated` or `superseded` (with a link)
Will do that instead.

Examples:
https://peterevans.dev/posts/lightweight-architecture-decision-records/
https://backstage.io/docs/architecture-decisions/adrs-overview
https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions

Comment (kitallis) [reply doc/arch/010-user-permissions.md:2]: Perfect 👍

**Pull request merged**